labconnect/main/serializers.py

```diff
@@ -0,0 +1,39 @@
+from flask import abort, render_template, request
+
+from labconnect import db
+from labconnect.helpers import SemesterEnum
+from labconnect.models import (
+    ClassYears,
+    Courses,
+    LabManager,
+    Leads,
+    Majors,
+    Opportunities,
+    RecommendsClassYears,
+    RecommendsCourses,
+    RecommendsMajors,
+    RPIDepartments,
+    RPISchools,
+)
+
+from . import main_blueprint
+
+
+def serialize_opportunity(opportunity):
```

Comment:
are you not using the .to_dict() serialization?

Reply:
I am using .to_dict(). This is a very old function and file. I'll try to delete it now.